install/ppa_release.sh

```diff
@@ -11,8 +11,8 @@ VERSION=`cat share/version.txt`
 
 rm -rf deb_dist >/dev/null 2>&1
 python3 setup.py --command-packages=stdeb.command sdist_dsc
-cd deb_dist/onionshare-$VERSION
+cd deb_dist/hyperddome-$VERSION
```

Comment:
hyperdome*

Reply:
this file should have been removed - I don't think Micah would currently allow us to post to his personal PPA